cpp/arcticdb/util/error_code.hpp

```diff
@@ -80,8 +82,9 @@ inline std::unordered_map<ErrorCategory, const char*> get_error_category_names()
     ERROR_CODE(7000, E_INVALID_USER_ARGUMENT) \
     ERROR_CODE(7001, E_INVALID_DECIMAL_STRING)   \
     ERROR_CODE(7002, E_INVALID_CHAR_IN_SYMBOL) \
-    ERROR_CODE(8000, E_UNRECOGNISED_COLUMN_STATS_VERSION) \
+    ERROR_CODE(8000, E_UNRECOGNISED_COLUMN_STATS_VERSION)          \
     ERROR_CODE(9000, E_DECODE_ERROR)
```

Comment:
The escape is required for the project compile.
```suggestion
    ERROR_CODE(8000, E_UNRECOGNISED_COLUMN_STATS_VERSION) \
    ERROR_CODE(9000, E_DECODE_ERROR) \
```